f2/src/forms/PrivacyConsentInfoForm.js

```diff
@@ -1,109 +1,91 @@
-            >
-              {consentOptions.map((key) => {
-                return (
-                  <React.Fragment key={key}>
-                    <CheckboxAdapter name="consentOptions" value={key}>
-                      <Trans id="privacyConsentInfoForm.yes" />
-                      <A
-                        href={'/privacystatement?lang=' + i18n.locale}
-                        isExternal
-                      >
-                        <Trans id="privacyConsentInfoForm.linkOut" />
-                      </A>
-                      <Trans id="privacyConsentInfoForm.period" />
-                    </CheckboxAdapter>
-                  </React.Fragment>
-                )
-              })}
-            </FormArrayControl>
-            <NextAndCancelButtons
-              next={
-                fyiForm ? (
-                  <Trans id="fyiForm.nextPage1" />
-                ) : (
-                  <Trans id="privacyConsentInfoForm.nextPage" />
-                )
-              }
-              button={<Trans id="privacyConsentInfoForm.nextButton" />}
-            />
-          </Stack>
-        )}
-      />
-    </React.Fragment>
+    <Formik
+      initialValues={whetherConsent}
+      onSubmit={(values) => {
+        props.onSubmit(values)
+      }}
+      validationSchema={PrivacyConsentInfoFormSchema()}
+    >
+      {({ handleSubmit, handleChange, handleBlur, errors, submitCount }) => (
+        <Form onSubmit={handleSubmit}>
+          <Container>
+            <Row classname="form-question">
+              {Object.keys(errors).length > 0 && (
+                <ErrorSummary
+                  errors={createErrorSummary(errors)}
+                  submissions={submitCount}
+                  title={
+                    <Trans id="privacyConsentInfoForm.hasValidationErrors" />
+                  }
+                />
+              )}
+            </Row>
+            <Row className="form-question">
+              <ErrorMessage
+                name="privacyConsentInfoForm.warning"
+                component={Error}
+              />
+            </Row>
+            <Row className="form-section">
+              <Field
+                name="consentOptions"
+                label={
+                  <Trans id="privacyConsentInfoForm.yes.withExternalLink">
+                    <A
+                      color="#0000ff"
+                      target="_blank"
+                      href={'/privacystatement?lang=' + i18n.locale} //{'/privacystatement?lang=en'}
+                    />
+                  </Trans>
+                }
+                component={CheckBox}
+                value="privacyConsentInfoForm.yes"
+                onChange={handleChange}
+                onBlur={handleBlur}
+                type="checkbox"
+                id="consentOptions"
+              ></Field>
+            </Row>
+            <Row>
+              <NextCancelButtons
+                submit={<Trans id="privacyConsentInfoForm.nextButton" />}
```

Comment:
The next page differs in the FYI flow. Need to change the label depending on report type.